src/templates/toolkit/bet/explanation.html.tt

```diff
@@ -20,8 +20,8 @@
     image1.staysinout     = [ request.url_for('images/pages/trade-explanation/in-out-4.svg') ]
     image.updown          = [ request.url_for('images/pages/trade-explanation/up-down-1.svg') ]
     image1.updown         = [ request.url_for('images/pages/trade-explanation/up-down-2.svg') ]
-    image.spreads         = [ request.url_for('images/pages/get-started/spreadbet-long.png') ]
```

Comment:
I couldn't find this in appcache, the only related spreads entry I could find I deleted from appcache which is the following:
push @cache_files, $request->url_for("images/pages/get-started/spread_bet.svg");